tmd/Soma/Soma.py

```diff
@@ -1,17 +1,19 @@
 '''
 tmd class : Soma
 '''
+import copy
+import numpy as np
 
 
 class Soma(object):
     '''Class of neuron soma
     '''
-    import numpy as _np
+    # pylint: disable=import-outside-toplevel
     from tmd.Soma.methods import get_center
     from tmd.Soma.methods import get_diameter
```

Comment:
same here. Is there a need in scoping these imports? It looks like you always create these objects when tmd is used.